weather-app/src/app/header/theme-switcher/theme-switcher.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-theme-switcher',
+  templateUrl: './theme-switcher.component.html',
+  styleUrls: ['./theme-switcher.component.scss'],
+})
+export class ThemeSwitcherComponent implements OnInit {
+  constructor() {}
+  lightTheme = 'light';
+  darkTheme = 'dark';
+
+  value = this.lightTheme;
+  ngOnInit(): void {}
+
+  onToggle(value: string): void {
+    console.log(value);
```

Comment:
1) remove console
2) this.theme = value === this.lightTheme ? this.darkTheme:  this.lightTheme